server/app.js

```diff
@@ -1,20 +1,32 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import dotenv from 'dotenv';
-
+import cors from 'cors';
+import swaggerUI from 'swagger-ui-express';
+import routes from './routers/routers';
+import swagger from '../swagger.json';
 
 dotenv.config();
 
+
 const app = express();
 
 app.use(express.json());
 
 app.use(bodyParser.json());
 
+app.use('/', routes);
+
+app.use(cors());
+
+
+app.use("/api-docs", swaggerUI.serve, swaggerUI.setup(swagger));
```

Comment:
don't put any route in your app.js file
so all I expect is `app.use(router); ` in your app.js file